### ISSUE TITLE
TypeError: 'NoneType' object is not iterable

### ISSUE THREAD

**Issue by rayzchen** [readthedocs/sphinx-hoverxref#192]
No idea what causes it.

https://readthedocs.org/projects/pyunity/builds/17067435/

**Comment by humitos** [comment]
Hi @rayzchen. If your project was already working before today and building correctly, I'd say this is probably a bug in the latest release 1.1.0 that I've done today 😞 

I'd suggest you pining your `sphinx-hoverxref` dependency to `<1.1.0` for now to keep your docs building. I will try to fix this issue and release a new version in the following week.

**Comment by humitos** [comment]
Some context:

* Python 3.9
* Sphinx 4.5
* Exception:
```
  File "/home/docs/checkouts/readthedocs.org/user_builds/pyunity/envs/develop/lib/python3.9/site-packages/sphinx/events.py", line 94, in emit
    results.append(listener.handler(self.app, *args))
  File "/home/docs/checkouts/readthedocs.org/user_builds/pyunity/envs/develop/lib/python3.9/site-packages/hoverxref/extension.py", line 242, in missing_reference
    for objtype in env.get_domain(domain).objtypes_for_role(reftype):
TypeError: 'NoneType' object is not iterable
```

**Comment by humitos** [comment]
Hi @rayzchen! I just released `1.1.1`. Please, let me know if this release fixes the issue you were having. Thanks!
